packages/tre/src/storage2/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from "./blob";
+export * from "./keyvalue";
+export * from "./sql";
+export * from "./storage";
```

Comment:
This file exports a lot. Do consumers need access to all the exports, or is just `storage` enough?

Reply:
Sounds good!